textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/common/FuzzyTokenPredicate.java

```diff
@@ -40,33 +43,62 @@ public FuzzyTokenPredicate(String query, List<Attribute> attributeList, Analyzer
             this.tokens = Utils.tokenizeQuery(analyzer, query);
             this.computeThreshold();
             this.attributeList = attributeList;
-            this.fields = this.extractSearchFields();
+            this.extractSearchFields();
             this.luceneQuery = this.createLuceneQueryObject();
         } catch (Exception e) {
             e.printStackTrace();
             throw new DataFlowException(e.getMessage(), e);
         }
     }
-    
-    private String[] extractSearchFields() {
-    	return null;
+    public boolean getisSpanInformationAdded(){
+    	return this.isSpanInformationAdded;
+    }
+    public List<Attribute> getAttributeList() {
+    	return this.attributeList;
+    }
+    public int getThreshold() {
+    	return this.threshold;
+    }
+    public ArrayList<String> getQueryTokens(){
+    	return this.tokens;
+    }
+    private void extractSearchFields() {
```

Comment:
No I don't think so. I also wrote something similar to this. But I put them in a list instead of an array. I think keyword matcher also has something similar too.